vp/src/core/rv64/mem.h

```diff
@@ -40,13 +40,15 @@ struct CombinedMemoryInterface : public sc_core::sc_module,
 	sc_core::sc_time dmi_access_delay = clock_cycle * 4;
 	std::vector<MemoryDMI> dmi_ranges;
 
-	MMU &mmu;
+	MMU *mmu;
```

Comment:
Why was this changed to a pointer instead of a reference? is there a case without an MMU?